frontend/src/shared/helpers/tokenExpiration/tokenExpirationCheck.ts

```diff
@@ -0,0 +1,13 @@
+const MILISECONDS = 1000;
+const MINUTE = 60 * MILISECONDS;
+export const tokenExpirationCheck = (token: string): boolean => {
+  const base64Url: string = token.split('.')[1];
```

Comment:
Uzyjmy moze pod to jednak biblioteki? Bo i tak to pewnie copy paste ze stackovetflow